floodwatch/src/js/components/Compare.js

```diff
@@ -62,43 +86,84 @@ export class CompareContainer extends Component {
 
   async componentDidMount(): any {
```

Comment:
Right, because React expects componentDidMount to not be `async`. It's more technically correct to have componentDidMount actually return void but kick off a Promise chain in it. All of which is to say, you can maybe just strip the return annotation entirely ¯\_(ツ)_/¯